packages/config/src/ConfigurationEditorDrawerWidget/components/__snapshots__/ConfigurationEditor.test.js.snap

```diff
@@ -2795,7 +2795,7 @@ exports[`ConfigurationEditor drawer widget renders with defaults of the PileupTr
                         spellcheck="false"
                         style="margin: 0px; border: 0px; background: none; box-sizing: inherit; display: inherit; font-family: inherit; font-size: inherit; font-style: inherit; font-variant-ligatures: inherit; font-weight: inherit; letter-spacing: inherit; line-height: inherit; tab-size: inherit; text-indent: inherit; text-rendering: inherit; text-transform: inherit; white-space: inherit; word-break: inherit; position: absolute; top: 0px; left: 0px; height: 100%; width: 100%; resize: none; overflow: hidden; padding: 10px 10px 10px 10px;"
                       >
-                        function(feature) { return feature.get('name') || feature.get('id') }
+                        function(feature) { return feature.get('name') || feature.get('id') || feature.id() }
```

Comment:
I don't think feature.id(), which is an internal feature ID, externally displayed

Reply:
yes, it should definitely be undefined or empty string I think

Reply:
feature.id() is for internal feature IDs, such as fileOffset based ids, it's just not something users need to see